packages/connectivity/src/scp-cf/cache.spec.ts

```diff
@@ -27,8 +27,10 @@ const dummyToken = {
 // Cache with expiration time
 const cacheOne = new Cache<Destination>(300000);
 
-// Cache without expiration time
-const cacheTwo = new Cache<ClientCredentialsResponse>();
+// Cache with a long expiration time
+const cacheTwo = new Cache<ClientCredentialsResponse>(
+  365 * 24 * 60 * 1000 /* 1 year */
+);
```

Comment:
[pp] How about:
```suggestion
// Never expiring cache
const cacheTwo = new Cache<ClientCredentialsResponse>(0);
```

Reply:
Thanks, yes the previous `1 year` was legacy code that I changed to simulate a very long cache, but in this case keeping the old behavior with our new "0 as never expiring" is definitely better.